src/test/kotlin/no/nav/tilleggsstonader/sak/interntVedtak/Testdata.kt

```diff
@@ -530,6 +531,17 @@ object Testdata {
                     ),
                 gitVersjon = Applikasjonsversjon.versjon,
             )
+        val avslåttVedtak =
+            GeneriskVedtak(
+                behandlingId = behandlingId,
+                type = TypeVedtak.AVSLAG,
+                data =
+                    AvslagBoutgifter(
+                        årsaker = listOf(ÅrsakAvslag.MANGELFULL_DOKUMENTASJON, ÅrsakAvslag.RETT_TIL_UTSTYRSSTIPEND),
```

Comment:
```suggestion
                        årsaker = listOf(ÅrsakAvslag.MANGELFULL_DOKUMENTASJON, ÅrsakAvslag.RETT_TIL_BOSTØTTE),
```
`RETT_TIL_UTSTYRSSTIPEND` er ikke en gyldig avslagsårsak for boutgifter, så det er litt rart å ha det i testdataene.

Reply:
Egentlig burde kanskje backend ha validert at ikke vi kan sende inn f.eks. `MANGELFULL_DOKUMENTASJON` i AvslagBoutgifter 🤔